soc/raspberrypi/rpi_pico/rp2350/Kconfig

```diff
@@ -4,22 +4,26 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_RP2350
+	select HAS_RPI_PICO
+	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
+	select SOC_RESET_HOOK
+	select XIP
+
+config SOC_RP2350A_M33
```

Comment:
seems to be wrong commit, should be in previous soc commit

Reply:
@nordicjm better now?

Reply:
Perfect! Thanks